tests/test_rbuser_tostring.py

```diff
@@ -0,0 +1,80 @@
+"""RedBrick Test Module; Tests the to_string method of the rbuser module."""
+
+import unittest
+from useradm import rbuser
+
+
+class RBUserTestCase(unittest.TestCase):
+    """Test Case class for to_string method"""
+
+    def setUp(self):
+        """Call before every test case."""
+        self.member_one_attr = {'cn': 'Member One',
+                                'id': 15358462,
+                                'uid': 'memberOne',
+                                'bday': 12,
+                                'host': 'carbon',
+                                'year': 3,
+                                'gecos': 'new member',
+                                'byear': 2017,
+                                'bmonth': 2,
+                                'passwd': 'e1NTSEF9WlBURGtJdmZJSld0WUlrak9zdGxscXYySTRRVEhBYj=',
+                                'newbie': True,
+                                'course': 'CASE',
+                                'created': '2009-10-06 11:20:13',
+                                'altmail': 'memberOne@example.com',
```

Comment:
this doesnt really matter now and i also dont know how it checks but part of the code that checks for ldap issues checks that the alt mail is a dcu mail. maybe make the email `@example.dcu.ie`